proxy/deploy.go

```diff
@@ -48,44 +69,29 @@ func DeployFunction(fprocess string, gateway string, functionName string, image
 }
 
 // Deploy a function to an OpenFaaS gateway over REST
-func Deploy(fprocess string, gateway string, functionName string, image string,
-	registryAuth string, language string, replace bool, envVars map[string]string,
-	network string, constraints []string, update bool, secrets []string,
-	labels map[string]string, annotations map[string]string,
+func Deploy(
+	req requests.CreateFunctionRequest,
+	gateway string,
+	language string,
```

Comment:
Is `language` being used anywhere?

Reply:
It's actually not. I can remove it and revalidate.